README.md

```diff
@@ -0,0 +1,3 @@
+# fsc.export
```

Comment:
I moved the ``README.md`` (previously just ``README``) from the ``doc`` subfolder to the top-level, to have a minimal description on github. Do you know if there was a strong reason why we put it in ``doc`` initially?

Reply:
No reason comes to mind right now, change it and we'll revert if we remember ^^

Reply:
😂